spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestExpireSnapshotsAction.java

```diff
@@ -1025,7 +1027,7 @@ public void testExpireAction() {
   }
 
   @Test
-  public void testUseLocalIterator() {
+  public void testUseCollectAsList() {
```

Comment:
This isn't needed, we can just fix the testUseLocalIterator test and not keep another version. This test was modified when we changed our Action api and it just wasn't updated correctly.